cli/dcoscli/marathon/main.py

```diff
@@ -656,6 +656,7 @@ def _clean_up_resource_definition(properties):
         if v:
             if k in ["apps", "groups"]:
                 clean_properties[k] = [_clean_up_resource_definition(v[0])]
+            # remove properties added by marathon
```

Comment:
How's `remove non-user specified properties for validation since they aren't included in json-schema`?